src/d/d_menu_collect.cpp

```diff
@@ -5,15 +5,323 @@
 
 #include "d/d_menu_collect.h"
 #include "dolphin/types.h"
+#include "stdio.h"
+#include "d/d_com_inf_game.h"
+
+#include "weak_bss_936_to_1036.h" // IWYU pragma: keep
+#include "weak_data_1811.h" // IWYU pragma: keep
+
+static dMc_HIO_c g_mcHIO;
 
 /* 8019BE8C-8019BF14       .text __ct__9dMc_HIO_cFv */
 dMc_HIO_c::dMc_HIO_c() {
-    /* Nonmatching */
+    m46 = 40;
+    m48 = 10;
+    m4A = 120;
+    m4C = 100;
+    m54 = 4.0f;
+    m5C = 0xff;
+    m5D = 70;
+    m52 = 130;
+    m4E = 250;
+    m50 = 20;
+    m58.set(-1, 170, 0, 0);
```

Comment:
And here too, colors are unsigned
```suggestion
    m58.set(0xFF, 0xAA, 0x00, 0x00);
```